### DISCUSSION TITLE
Need help applying for Students Developer Pack

### DISCUSSION THREAD

**Discussion by davidmayr** [community/community#52181] [GitHub Education] [answered]
Hello,
I've had the pack for over 3 years now and never had an issue applying, but now my 3rd request today was denied. 
Previously I've used my personal email, and so I did for the first request today too. Since it got denied, I tried it with a school issued email again twice, but was still denied.

In all 6 requests (the three that got denied today and the three that worked the last 3 years) I've ever issued, I used my school ID called "edu.card" in Austria. In my last request, I added a school notice that gets sent out when one semester ends and the next one begins, but the webcam quality was so bad I uploaded a photo I took with my phone and transferred to my pc. That request was denied with this message:" Please consider using your device camera to submit academic affiliation documents. Uploaded images are more easily manipulated and are therefore less trustworthy."

All the other requests were denied without reason and with a guide to common verification issues, but I don't see why I wouldn't get the pack. Especially since I got it the last 3 years. 

One thing I might add is that, if I recall correctly, I previously had my school email attached to my account and then removed again. Now it's linked again, though. I don't see why that should be an issue, but I might as well add it.

**Comment by davidmayr** [comment, marked as answer]
I tried resubmitting on my phone and it worked now. Only took 4 requests 😅

**Reply by benjalonc** [reply]
Thanks, worked after 4 requests using my phone!
